src/support.py

```diff
@@ -278,8 +283,18 @@ def draw_aa_line(
         - (length / 2.0) * math.sin(deg),
     )
 
-    pygame.gfxdraw.aapolygon(surface, (ul, ur, br, bl), color)
-    pygame.gfxdraw.filled_polygon(surface, (ul, ur, br, bl), color)
+    pygame.draw.aalines(
+        surface=surface,
+        color=color,
+        closed=True,
+        points=(ul, ur, br, bl),
+    )
+    pygame.draw.polygon(
+        surface=surface,
+        color=color,
+        points=(ul, ur, br, bl),
+        width=0,  # width=0 -> filled
+    )
```

Comment:
I'm curious about this change. Does pygbag not work with `pygame.draw.aapolygon` or `pygame.draw.filled_polygon`?

Reply:
pygame.gfxdraw has problems on pygbag indeed.

Reply:
Ah, I see. Could we just check for pygbag then, and select the function based on the platform?

Reply:
@fortwoone I wouldn't recommend that, because then we might have inconsistencies between local vs. web runs, making things harder to build and debug. It would also mean that developers needs to keep track of 2 different implementations, remember to add `if` statements to check the platform.

I think it's better, especially for core graphics calls, if we pick the APIs that will be generally compatible with local and web, and keep it consistent.